src/styles/app.scss

```diff
@@ -1,9 +1,18 @@
+$mainBackgroundColor: #e1f3f8;
+// @import './settings.scss';
+
 main {
-  padding-bottom: 200px;
-  background-color: #e1f3f8;
+  padding-bottom: 250px;
+  // background-color: $theme-color-primary-light;
```

Comment:
For some reason using the $theme-color-primary-light variable doesn't seem to work for me (even when I import settings.scss on line 2). I realized that I haven't been compiling my scss files. Do I need to be doing so in a create-react-app?

Reply:
I have also been stuck on SASS styling, as indicated in my last PR, so I won't be of much help. :/